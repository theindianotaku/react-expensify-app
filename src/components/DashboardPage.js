import React, { Component } from 'react';

import ExpenseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';
import ExpenseSummary from './ExpenseSummary';

class DashboardPage extends Component {
  render() {
    return (
      <div>
        <ExpenseSummary />
        <ExpenseListFilter />
        <ExpenseList />
      </div>
    );
  }
}

export default DashboardPage;
