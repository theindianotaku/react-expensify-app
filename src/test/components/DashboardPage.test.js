import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('should render App component', () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
