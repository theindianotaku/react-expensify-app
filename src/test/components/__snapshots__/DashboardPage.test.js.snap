// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component 1`] = `
<div
  className="m-b-l"
>
  <Connect(Component) />
  <Connect(ExpenseListFilter) />
  <Connect(Component) />
</div>
`;
